configure

```diff
@@ -0,0 +1,212 @@
+#!/bin/bash
+
+function checkCompiler {
```

Comment:
can you use underscore case instead of camlCase please 
we use Linux style which use underscore in C, so why change it for shell script ?